src/test/Player.gd

```diff
@@ -1,7 +1,75 @@
 extends Sprite
 
-onready var sprite_manager_ref = get_node(NodePath("/root/Sample/SpriteManager"))
+onready var sprite_manager_ref: Node = get_node(NodePath("/root/Sample/SpriteManager"))
+
+const player_left_rect = Rect2(0, 0, 8, 8)
+const player_left2_rect = Rect2(0, 8, 8, 8)
```

Comment:
We're probably not going to want to define the type `const player_left2_rect: Rect2 = Rect2(0, 8, 8, 8)` because it is obviously Rect2 (in c++/c# we would use auto/var)